packages/boxel-motion-test-app/tests/helpers/resolve-generator.ts

```diff
@@ -0,0 +1,21 @@
+import { Frame } from '@cardstack/boxel-motion/behaviors/base';
+
+export default function resolveGenerator(generator: Generator<Frame | void>) {
+  let frames: (Frame | undefined)[] = [];
+  let next = generator.next();
+  while (!next.done) {
+    let { value, velocity } = next.value as Frame;
+    let frame;
+    if (next.value) {
+      frame = {
+        value,
+        velocity,
+      };
+    }
+    frames.push(frame);
```

Comment:
Should this be inside the `if` block, so that we don't accidentally push `undefined`?